src/main/java/racingwinner/Car.java

```diff
@@ -0,0 +1,15 @@
+package racingwinner;
+
+public class Car {
+    String name;
```

Comment:
인스턴스 변수의 접근 제한자는 private으로 구현합니다.

```suggestion
    private String name;
```